Engine/debug/debugger.h

```diff
@@ -46,8 +46,15 @@ void check_debug_keys();
 #define DBG_REGONLY   0x200
 #define DBG_NOVIDEO   0x400
 
+enum FPSDisplayMode
+{
+    kFPS_Hide = 0,    // hid by the script/user command
+    kFPS_Display = 1, // shown by the script/user command
```

Comment:
To avoid confusion, I suggest commenting out kFPS_Display until it is actually supported by a script command.